src/components/Modal/BaseModal.tsx

```diff
@@ -180,7 +180,7 @@ function BaseModal(
             onModalHide={hideModal}
             onModalWillShow={() => ComposerFocusManager.resetReadyToFocus()}
             onDismiss={handleDismissModal}
-            onSwipeComplete={onClose}
+            onSwipeComplete={() => onClose()}
```

Comment:
Why?
```suggestion
            onSwipeComplete={onClose}
```

Reply:
That won't work, because `onClose` has an optional **ref** argument that is not supported by `onSwipeComplete`

Reply:
Ah, okay.

Reply:
oh, this was existing bug on production?